src/fragments/gen2/quickstart/build-a-backend.mdx

```diff
@@ -151,7 +151,7 @@ import { defineAuth } from '@aws-amplify/backend';
 /**
  * Define and configure your auth resource
  * When used alongside data, it is automatically configured as an auth provider for data
- * @see https://docs.amplify.aws/gen2/build-a-backend/auth
```

Comment:
Pretty sure this is autogenerated code by the gen2 CLI. Can you doublecheck with the CLI team? We'll want to get these updated in the CLI itself at launch.

Reply:
to echo @reesscot , it is! https://github.com/aws-amplify/amplify-backend/blob/b0112e34f94b05179addf67c213a5cfce16dc079/packages/create-amplify/templates/basic-auth-data/amplify/auth/resource.ts#L6

Reply:
Got it, I'll change it back to how it originally was then